install/kubernetes/helm/meshery/charts/mesheryconsul/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mesheryconsul.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "mesheryconsul.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "mesheryconsul.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "mesheryconsul.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "mesheryconsul.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          # TODO: Need to confirm the mesheryistio image version
```

Comment:
mesheryconsul

Reply:
Good catch. Yes, this tag needs to be `stable-latest`. I suspect this update is needed across all other charts, too.

Reply:
Updated